<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Screen_Log</class>
 <widget class="QMainWindow" name="Screen_Log">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(66, 198, 198);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Button_Log">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>310</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 255, 127);
font: 25pt &quot;Niagara Solid&quot;;
color: rgb(0, 0, 0);
border-radius:5px;</string>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Input_User">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>401</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(241, 241, 241);
border-style: groove;
border-width: 3px;
border-radius:15px;
border-color: rgb(0,0,0);
font: 20pt &quot;Times New Roman&quot;;
</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Email</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Input_Key">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>190</y>
      <width>401</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(241, 241, 241);
border-style: groove;
border-width: 3px;
border-radius:15px;
border-color: rgb(0,0,0);
font: 20pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Senha</string>
    </property>
   </widget>
   <widget class="QLabel" name="Icon_User">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>assets/person.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Icon_Key">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>assets/lock.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Ornament_Welcome">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>391</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;Niagara Solid&quot;;
font: 20pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>BEM VINDO</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_Reg">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>430</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 255, 127);
font: 25pt &quot;Niagara Solid&quot;;
color: rgb(0, 0, 0);
border-radius: 5px;</string>
    </property>
    <property name="text">
     <string>Registrar-se</string>
    </property>
   </widget>
   <widget class="QLabel" name="Ornament_Question">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;Niagara Solid&quot;;
font: 20pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>NÃ£o possui uma conta?</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Warning_Wrong_Credencial">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>461</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);
font: 20pt &quot;Niagara Solid&quot;;
font: 20pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
